oz_tree_build/images_and_vernaculars/get_wiki_images.py

```diff
@@ -190,10 +188,10 @@ def get_vernaculars_by_language_from_json_item(json_item):
 
     # For each language:
     # - We keep all the vernaculars
-    # - If none are marked as preferred, the first non-referred will be marked as preferred
+    # - If none are marked as preferred, use the first non-preferred as preferred
     # - If multiple are marked as preferred, the first one will be kept as preferred
     for vernaculars in vernaculars_by_language.values():
-        vernaculars.sort(reverse = True, key = lambda v: v["preferred"])
+        vernaculars.sort(reverse=True, key=lambda v: v["preferred"])
```

Comment:
This might be another configurable thing, if we wanted to stay closer to Black?

Reply:
I'm not bothered. I prefer the tight equals (no spaces) in parameter settings, and ISTR that black used to do this too?

Reply:
Ok, I'm fine with changing it, it was more about reducing the churn. If we temporarily move back to Black to reduce to diff (per other discussion), we can later use the Ruff defaults.